File: js/Salario.js
let ingreso = 3500000
if (ingreso < 1500000) {
    console.log("Sin impuesto")
    console.log("Valor neto", ingreso);
}
else if (ingreso >= 1500000 && ingreso <= 3000000) {
    console.log("Impuesto del 10%")
    console.log("Valor neto", ingreso * 0.90);
}
else {
    console.log("Impuesto de 20%")
    console.log("Valor neto", ingreso * 0.80)
}

File: py/salario.py
ingresos = int(input("Poner sus ingresos: "))

if ingresos < 1500000:
    print ("Sin impuestos")
    print ("Valor neto", ingresos)
elif ingresos >= 1500000 and ingresos <= 3000000:
    print ("Impuesto del 10%")
    print ("Valor neto", ingresos * 0.90)
else:
    print("Impuesto del 20%")
    print ("Valor neto", ingresos * 0.60)
    